docs/REST API/rest-api.md

```diff
@@ -8943,6 +8945,43 @@ Request (using `filters`):
 }
 ```
 
+### Get batch script summary
+
+Get statuses and host counts for a batch-run script.
+
+`GET /api/v1/fleet/scripts/batch/summary`
```

Comment:
Per [agreement with @sgress454](https://github.com/fleetdm/fleet/pull/29200/files#r2093811935) that this should be a path parameter
```suggestion
`GET /api/v1/fleet/scripts/batch/summary/:batch_execution_id`
```